### PR TITLE
docs: migrating-to-renku, templates, dockerfiles

### PR DESCRIPTION
**Pull request by emmjab**
Documentation for
* Migrating a project to renku
* The structure of directories and files provided by the templates
* How to customize the Dockerfile
* A starting point for documentation on how to document a data science project on renku

### CODE REVIEW

File: docs/user/templates.rst [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+ README.md
+ environment.yml
+ requirements.txt
+
+data and notebooks
+""""""""""""""""""
+
+The ``data`` and ``notebooks`` directories are where you should keep your...
+data (added via ``renku dataset``) and notebooks (by double-clicking on a python3
+or R kernel image from the jupyterlab instance while you're inside the dir), respectively.
```
Comment (fgeorgatos) [inline docs/user/templates.rst]: `... data` -> `data` ?

`dir)` -> `directory`

File: docs/user/migrating_to_renku.rst [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+  $ git remote add renku git@renkulab.io/<namespace>/<project-name>.git
+  $ git push renku master
+
+View your project on Renku web
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If everything went smoothly, you should be able to view your project at
+https://renkulab.io/gitlab/<namespace>/<project-name>. When you push code to this
+repo, the CI/CD pipeline will run according to settings in the generated ``gitlab-ci.yml``
```
Comment (fgeorgatos) [inline docs/user/migrating_to_renku.rst]: `.gitlab-ci.yml` (dot)

File: docs/user/migrating_to_renku.rst [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+file, rebuilding your docker image as specified by the generated ``Dockerfile``.
+If your build fails, see :ref:`advanced_interfaces`.
+
+.. warning::
+
+  If you already had a ``Dockerfile`` and/or ``gitlab-ci.yml`` file, keep a close
```
Comment (fgeorgatos) [inline docs/user/migrating_to_renku.rst]: ditto `.gitlab-ci.yml`

**Comment by rokroskar** [comment]
closing, duplicate of #644

**Pull request closed without merging**